hack/testing/rsyslog/roles/rsyslog/tasks/main.yaml

```diff
@@ -9,7 +9,7 @@
     - '{{ rsyslog__viaq_packages if rsyslog__viaq|bool else [] }}'
     - '{{ rsyslog__tls_packages if rsyslog__pki|bool else [] }}'
     - '{{ rsyslog__packages }}'
-  when: rsyslog__enabled|bool
+  when: rsyslog__enabled|bool and use_rsyslog_image|bool
```

Comment:
sorry, this should be `not use_rsyslog_image|default(False)|bool` - that is, if you are using the image, you do not want to install the rsyslog packages such as rsyslog, rsyslog-mmnormalize, etc.